pkg/api/customization/persistentvolumeclaim/validator.go

```diff
@@ -27,10 +28,14 @@ func (v *Validator) Validator(request *types.APIContext, schema *types.Schema, d
 	}
 
 	storageClass, err := c.Storage.StorageClasses("").Get(storageClassID, v1.GetOptions{})
-	if err != nil {
+	if err != nil && !apierrors.IsNotFound(err) {
```

Comment:
A nil chick on the `storageClass` isn't needed, if there is an error process accordingly, otherwise we assume we have a real thing. 

```suggestion
	if err != nil {
		if !apierrors.IsNotFound(err) {
			return err
		}
		return nil
	}
```